src/main/java/com/zendesk/maxwell/row/RowIdentity.java

```diff
@@ -0,0 +1,107 @@
+package com.zendesk.maxwell.row;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.*;
+
+public class RowIdentity implements Serializable {
+	private String database;
+	private String table;
+	private final List<Map.Entry<String, Object>> primaryKeyColumns;
+
+	// slightly less verbose pair construction
+	public static Map.Entry<String, Object> pair(String key, Object value) {
+		return new AbstractMap.SimpleImmutableEntry<>(key, value);
+	}
```

Comment:
elsewhere we use `org.apache.commons.lang3.tuple.Pair`, which is probably cleaner than all of this...